src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+  background-color: white;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500px;
+
+  --accent-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo__img {
+  cursor: pointer;
+}
+
+.nav__menu {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
```

Comment:
I recommend moving all `reset type` styles to the top, later they all will lay in `reset.css`, so get used to separating